esp-hal/src/i2c/master/mod.rs

```diff
@@ -467,7 +467,10 @@ impl<Dm: DriverMode> embedded_hal::i2c::I2c for I2c<'_, Dm> {
     }
 }
 
-impl<'d, Dm: DriverMode> I2c<'d, Dm> {
+impl<'d, Dm> I2c<'d, Dm>
+where
```

Comment:
Just out of curiosity, why was this change made? On some other places we don't use `where`, for example:
`pub struct Spi<'d, Dm: DriverMode> {`